### ISSUE TITLE
Lexing of strings returns BadToken

### ISSUE THREAD

**Issue by zlikavac32** [axmachado/pacc#3]
`footer` can exist as an option with provided string value but string escape sequence do not work

`footer = "$var = \"string\";";` throws `Unexcepted token \`

Is this considered a bug? I know that ' can be used instead of " etc or @inner.

I have a version of Regex that works, but it seams that `StringToken` will have to be changed as well.

I could see about this one as well

**Comment by axmachado** [comment]
You shoud use:

~~~yacc
@footer {
    $var = "string";
}
~~~

instead of 
~~~yacc
footer= "$var \"string\";";
~~~

**Comment by zlikavac32** [comment]
Yes, that's what I'm using now. Should that be documented? Because you can't even use \  like this

```
footer = "/* \ abcd \ */";
```

This removes \ and final result is `/*  abcd  */`

**Comment by axmachado** [comment]
The `footer = ""` syntax should be removed. I will mark it as a bug, because it will never understand complex expressions using this syntax without implementing a big portion PHP grammar to eval what is inside the `()`.
